src/Components/Navbar/nav.module.css

```diff
@@ -0,0 +1,10 @@
+.nav{
```

Comment:
I think there's a problem with indentation - don't forget to press "Shift + Alt + F".
If it doesn't work by default for "css" files - consider installing the extension for VS Code.

Reply:
i added eslint config and prettier. And now i can use this combination